src/components/ModelTableList/shared.js

```diff
@@ -129,3 +134,76 @@ export function getStatusValue(status, key) {
   }
   return returnValue;
 }
+
+/**
+  Generates the cloud and region info from model data.
+  @param {Object} model The model data.
+  @returns {Object} The React element for the model cloud and region cell.
+*/
+export const generateCloudCell = (model) => {
```

Comment:
Looks like this file uses the style

`export function foo() {}` would you mind updating these to be consistent with the rest of the file?